cdap-distributions/src/emr/install.sh

```diff
@@ -22,9 +22,9 @@
 # The git branch to clone
 CDAP_BRANCH=${CDAP_BRANCH:-release/4.0}
 # Optional tag to checkout - All released versions of this script should set this
-CDAP_TAG='v4.0.1'
+CDAP_TAG='v4.0.2-SNAPSHOT'
```

Comment:
This will break EMR installation. This is the *RELEASE* tag. We don't tag snapshots.